### DISCUSSION TITLE
Add docker Support

### DISCUSSION THREAD

**Discussion by DanBrown47** [ashishpapanai/chessJS#103] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not related to a problem but an enhancement

**Describe the solution you'd like**
Adding a docker support will help contributors to test easily and deployment to cloud will become more simpler

**Describe alternatives you've considered**
Adding Dockerfile for docker support

**Additional context**
This would enable easy deployment and orchestration to cloud

**Comment by ashishpapanai** [comment]
Do you want to work on this issue @DanBrown47?

**Comment by DanBrown47** [comment]
> Do you want to work on this issue @DanBrown47? 

Yes I want to

**Comment by ashishpapanai** [comment]
Please start working on it @DanBrown47.
